src/routes.js

```diff
@@ -74,89 +74,14 @@ const router = async app => {
 	 * Get yourself a list of repositories.
 	 */
 	router.get("/repositories", async (req, res) => {
+		let repositoryList = repositories.list().map(({name, topics}) => ({name, topics}));
 		if (req.query.topic) {
-			await handleFiltered(req, res);
-		} else {
-			handleDefault(req, res);
-		}
-	});
-
-	/**
-	 * Handle the default route for /repositories.
-	 *
-	 * @param {import('express').Request} req
-	 * @param {import('express').Response} res
-	 */
-	const handleDefault = (req, res) => {
-		const repositories = Array.from(repositoryStore).map(
-			// eslint-disable-next-line no-unused-vars
-			([key, repository]) => ({
-				name: repository.name
+			repositoryList = repositoryList.filter(repository => {
+				return repository.topics.includes(req.query.topic)
```

Comment:
Humm. Something to fix, and a a test case to cover this error.

Do all repositories have topics?

```
TypeError: Cannot read property 'includes' of undefined
  at repositoryList.filter.repository (/app/src/routes.js:80:30)
  at Array.filter (<anonymous>)
```